src/util/query.js

```diff
@@ -105,7 +102,10 @@ module.exports.build = (allowedFields, {
         boost_mode: 'replace'
       }
     };
-    result.sort.push({ _score: { order: 'desc' } });
   }
+  result.sort = orderBy
+    .concat(scoreBy.length !== 0 ? [['_score', 'desc', null]] : [])
+    .concat(isEqual(orderBy.slice(-1), [['id', 'asc']]) ? [] : [['id', 'asc']])
```

Comment:
Can we use destructuring here? I hate using concat with arrays... It's never clear to see how things get resolved...